pkg/executor/api.go

```diff
@@ -249,6 +249,62 @@ func (executor *Executor) unTapService(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusOK)
 }
 
+// isValidHandler is very similar to getServiceForFunctionAPI, with the difference that the response returned doens't contain
+// the address, but instead returns "true" or "false", depending if the function service is valid or not.
+func (executor *Executor) isValidHandler(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	logger := otelUtils.LoggerWithTraceID(ctx, executor.logger)
+
+	body, err := io.ReadAll(r.Body)
+	if err != nil {
+		http.Error(w, "Failed to read request", http.StatusInternalServerError)
+		return
+	}
+
+	// get function metadata
+	fn := &functionWithAddress{}
+	err = json.Unmarshal(body, &fn)
+	if err != nil {
+		logger.Error("Error unmarshelling: ", zap.Any("error", err))
```

Comment:
spelling mistake, *unmarshalling

Reply:
ehehe, sorry about that. Just fixed it! Please take a look!